### DISCUSSION TITLE
Is the Gateway Operator Open Source

### DISCUSSION THREAD

**Discussion by frankgreco** [Kong/kong#12156] [Help] [answered]
I'm having issues getting it working so I thought I'd debug using the code. However, i'm having trouble finding the codebase on GitHub.

**Comment by chobits** [comment]
Bring in the big guns! hi @randmonkey could you provide some information in this discussion thread, ty!

**Reply by chobits** [reply]
hi @frankgreco I have examined our internal resources, finding that KGO project is closed-source currently.

hi @mheap , this discussion is around the KGO project. If necessary, you can provide additional information about the KGO project. Thank you!

**Reply by frankgreco** [reply]
Is the plan to open source it? I can't imagine it would contain a bunch of IP given that it's just an operator. What is the general approach to filing bugs, getting help, etc. for that project?

Even if it did, "source available" with a restrictive Kong license would be nice too.

**Reply by mflendrich** [reply]
Responded in https://github.com/Kong/gateway-operator-docs/discussions/29#discussioncomment-7774222

**Comment by mflendrich** [comment, marked as answer]
Duplicate of https://github.com/Kong/gateway-operator-docs/discussions/29
